### ISSUE TITLE
Call Chat and Language Model APIs?

### ISSUE THREAD

**Issue by thorwebdev** [sourcegraph/openctx#171]
I'm trying to create an experience similar to https://marketplace.visualstudio.com/items?itemName=robconery.pg-chat 

Vscode allows you to run prompt's in the users Copilot account like this https://code.visualstudio.com/api/extension-guides/language-model 

Is this something that's possible for Cody also?

**Comment by thorwebdev** [comment]
Another thing that would be great, would be the ability to generate embeddings for the user queries. E.g. with the [Postgres Provider](https://github.com/sourcegraph/openctx/pull/175) I'm working on, would be awesome to do a cosine similarity search to retrieve all the relevant context from the Postgres database using pgvector.

**Comment by sqs** [comment]
No, neither of these are possible now with Cody, although you can use `require('vscode')` in an OpenCtx provider and tap into the entire VS Code extension API, including Copilot's language model API.
